### PR TITLE
A couple map funcs

### PR DESCRIPTION
**Pull request by ethteck**
And some others

### CODE REVIEW

**Comment by BowserSlug** [comment]
🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀 This PR matches 193 functions (+0.92%) on `us`.

**Comment by Brotenko** [comment]
Next time please refrain from opening a PR unless you have a few more functions for us to review, thanks!
Otherwise, very good work 🙏

File: src/world/area_jan/jan_00/jan_00_2_npc.c [hunk 1]
```diff
@@ -98,17 +98,71 @@ EvtScript N(EVS_Kolorado_LetterReward) = {
...
+API_CALLABLE(func_80240B4C_B2108C) {
+    Bytecode* args = script->ptrReadPos;
+    Npc* npc;
+    f32 temp_f20;
+    f32 var_f22;
+    f32 x, y, z;
+    s32 temp_s1;
+    s32 outX, outY, outZ;
+
+
```
Comment (nanaian) [inline src/world/area_jan/jan_00/jan_00_2_npc.c:113]:
```suggestion
```

File: src/world/area_pra/pra_33/D89700.c [hunk 2]
```diff
@@ -2,16 +2,30 @@
 
 #include "world/common/atomic/Reflection.inc.c"
 
-INCLUDE_ASM(s32, "world/area_pra/pra_33/D89700", func_80240F20_D8A600);
+ApiStatus func_80240F20_D8A600(Evt* script, s32 isInitialCall) {
+    script->varTable[10] = 0;
+    if (gCollisionStatus.currentFloor == 0x1C) {
+        script->varTable[10] = 1;
+    }
+    if (gCollisionStatus.lastTouchedFloor == 0x1C) {
```
Comment (nanaian) [inline src/world/area_pra/pra_33/D89700.c:10]: Can you use the COLLIDER_x macros from pra_33_hit.h?

**Review by nanaian — CHANGES_REQUESTED** [review]
insane

**Pull request merged**